src/Base32.js

```diff
@@ -88,13 +86,10 @@ class Base32 {
   }
 }
 
-Base32.DIGITS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ234567'.split('')
+Base32.DIGITS = [...'ABCDEFGHIJKLMNOPQRSTUVWXYZ234567']
```

Comment:
Based on my testing `string.split('')` is significantly faster than the spread syntax

Reply:
I'm not sure what means significantly, but on my computer the function wil take the 0.0001 vs 0.0002 ms per call.
Okay if we loop this 10000 times we will loose 1ms.

Reply:
The `string.split()` will not correctly handle unicode characters by the way.
If you wan't pick the parts you need.

Reply:
> I'm not sure what means significantly, but on my computer the function will take 0.0001 vs 0.0002 ms per call.
> Okay if we loop this 10000 times we will loose 1ms.

That's my fault. I tested it with a much longer string. The spread syntax will perform significantly worse the longer string is, while the split function performs pretty much the same.

> The `string.split()` will not correctly handle unicode characters by the way.

Not a problem in this case.

***

Either way, I see no point in this change.